source/languages/en/riak/dev/using/basics.md

```diff
@@ -88,6 +88,12 @@ obj = bucket.get('rufus')
 var id = new RiakObjectId("animals", "dogs", "rufus");
 ```
 
+```javascript
+client.fetchValue({ bucketType: 'animals', bucket: 'dogs', key: 'rufus' }, function (err, rslt) {
+    // rslt.isNotFound === true since rufus has not been saved
```

Comment:
I would write this like...
```
`rslt.isNotFound === true` since `rufus` has not been saved.
```
just to help differentiate pseudocode and prose within the comment.